src/test/ui/proc-macro/proc-macro-deprecated-attr.rs

```diff
@@ -0,0 +1,15 @@
+// build-pass
```

Comment:
```suggestion
// check-pass
```
It's necessary to do codegen/linking to check deprecation.